### DISCUSSION TITLE
Starstruck should apply to repos you're a contributor to, rather than ones you create.

### DISCUSSION THREAD

**Discussion by wiki-Bird** [community/community#20642] [Other Feature Feedback, Questions, & Ideas]
If you're a major contributor to a starred repository, I don't think it makes sense than an owner account gets the achievement yet you don't. There are repos I've worked on (and not owned) where I've done more work than the account which owns it, and repos I've owned in which other people have done more work than I.

Because of this, I think starstruck should apply to repos you're a contributor to, rather than ones you create.

**Comment by devkennyy** [comment]
What about a minor typo contribution? I don't think they deserve the same recognition as the maintainer/creator of the repo?
